### ISSUE TITLE
bleachbit_3.0_all_ubuntu1404.deb fails to install on Ubuntu 14.04

### ISSUE THREAD

**Issue by assarbad** [bleachbit/bleachbit#694]
Hi,

I didn't find a matching (existing) issue, so I am opening one. I realize that this LTS version of Ubuntu is no longer officially supported, but since the package is provided, I thought I'd drop a line.

When attempting the installation I see:

```
$ dpkg -i bleachbit_3.0_all_ubuntu1404.deb
Selecting previously unselected package bleachbit.
(Reading database ... 474615 files and directories currently installed.)
Preparing to unpack bleachbit_3.0_all_ubuntu1404.deb ...
Unpacking bleachbit (3.0) ...
dpkg: dependency problems prevent configuration of bleachbit:
 bleachbit depends on python-scandir; however:
  Package python-scandir is not installed.

dpkg: error processing package bleachbit (--install):
 dependency problems - leaving unconfigured
Processing triggers for desktop-file-utils (0.22-1ubuntu1.1) ...
Processing triggers for mime-support (3.54ubuntu1.1) ...
Processing triggers for gnome-menus (3.10.1-0ubuntu2) ...
Errors were encountered while processing:
 bleachbit
```

However, the usual `apt install` to resolve them doesn't work:

```
$ apt install 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
You might want to run 'apt-get -f install' to correct these.
The following packages have unmet dependencies:
 bleachbit : Depends: python-scandir but it is not installable
E: Unmet dependencies. Try using -f.
```

The recommended (by above error message) `apt install -f` ends up removing the half-installed `bleachbit` package.

Also `apt-cache search scandir` comes back empty.

System is:

```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.6 LTS
Release:	14.04
Codename:	trusty
```

Thanks.

**Comment by assarbad** [comment]
**A temporary workaround** is the following (as superuser):

```
apt install python-pip
pip install scandir
dpkg --ignore-depends=python-scandir -i bleachbit_3.0_all_ubuntu1404.deb
```

**Comment by Francesco-Cocchioni** [comment]
Same problem also on Ubuntu 16.04. 

The package python-scandir is available only from Ubuntu 18.04:

https://launchpad.net/ubuntu/+source/python-scandir

**Comment by az0** [comment]
This is documented in the [known issues section of the release notes](https://www.bleachbit.org/news/bleachbit-30#known), and it's a duplicate of the fixed issue https://github.com/bleachbit/bleachbit/issues/682

Here is a pre-release that has a fix: [Ubuntu 14.04](http://download.opensuse.org/repositories/home:/andrew_z/xUbuntu_14.04/all/), [Ubuntu 16.04](http://download.opensuse.org/repositories/home:/andrew_z/xUbuntu_16.04/all/bleachbit_3.0.1_all.deb)

If this doesn't work, let me know.

**Comment by az0** [comment]
@assarbad 
>I realize that this LTS version of Ubuntu is no longer officially supported, but since the package is provided, I thought I'd drop a line.

According to https://wiki.ubuntu.com/Releases , Ubuntu 14.04 (Trusty Tahr) is supported through April 2022, so I will try to keep supporting it.

**Comment by assarbad** [comment]
> According to https://wiki.ubuntu.com/Releases , Ubuntu 14.04 (Trusty Tahr) is supported through April 2022, so I will try to keep supporting it.

Oh, fascinating. I was under the impression that it used to be five years ... sorry!

**Comment by mhampl** [comment]
> > According to https://wiki.ubuntu.com/Releases , Ubuntu 14.04 (Trusty Tahr) is supported through April 2022, so I will try to keep supporting it.
> 
> Oh, fascinating. I was under the impression that it used to be five years ... sorry!

"Standard" support for Ubuntu 14.04 (Trusty Tahr) has already ended in April 2019, but it currently is in ESM (Extended Security Maintenance), which is a paid option for selected server packages, see https://ubuntu.com/esm and https://wiki.ubuntu.com/SecurityTeam/ESM/14.04

**Comment by az0** [comment]
I published this fix in [BleachBit 3.1.0 beta](https://www.bleachbit.org/news/bleachbit-31-beta). Perhaps you could test and write back here with any comments about how it addresses your issue.
